oak_functions/abi/src/lib.rs

```diff
@@ -47,12 +47,12 @@ pub mod proto {
         }
     }
 
-    // Converts a ChannelStatus into an OakStatus.
-    impl From<OakStatus> for ChannelStatus {
-        fn from(oak_status: OakStatus) -> Self {
-            match oak_status {
-                OakStatus::ErrInvalidArgs => ChannelStatus::ChannelInvalidArgs,
-                _ => ChannelStatus::Unspecified,
+    // Converts a OakStatus into an ChannelStatus.
+    impl From<ChannelStatus> for OakStatus {
```

Comment:
I know this will go away at some point, but it seems backwards to me. Should we not have the functions return ChannelStatus, and convert to OakStatus?

Reply:
Which functions should return a ChannelStatus?

Reply:
Never mind, I actually read the proposed change the other way round, this makes sense. :)